### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by kayGweb** [Abhinandan-Kushwaha/react-native-gifted-charts#279] [Q&A]
I have a question about the capabilities of your repository as we are using it in an upcoming public-facing application.  

So as you suggested, let me introduce myself.  I am Kaygweb online; however, my name is Ideal Kendall.  I am a software engineer working with mostly .Net learning firms with my strength in C# and JavaScript; in addition, I have a database background.  

Now I would like to speak about "getProinterProps" function.  Do you have an example of how this is used?  And is this the best way to get the current pointer location?

Looking forward to chatting with everyone.

Thanks,

**Comment by Abhinandan-Kushwaha** [comment]
Hi @kayGweb 👋
I am replying so late, most probably you would have moved on. By any chance, if you are still working, I would be more than delighted to discuss.
